src/winforms/toga_winforms/widgets/imageview.py

```diff
@@ -0,0 +1,34 @@
+from toga_winforms.libs import *
+
+from .base import Widget
+
+
+class ImageView(Widget):
+
+    def create(self):
+        self.native = WinForms.PictureBox()
+        self.native.interface = self.interface
+        self.native.BorderStyle = WinForms.BorderStyle.Fixed3D
+
+    def get_image(self):
+        return self.native.image
+
+    def set_image(self, image):
+        if image:
+            # Workaround for loading image from url
+            if isinstance(image._impl.native, str):
+                self.native.Load(image._impl.native)
+            else:
+                self.native.Image = image._impl.native
+        else:
+            # When do we have no image here???
```

Comment:
Consider the case of an image browser - there's a list of image files that can be displayed, and an image view. By default, no image is selected, so no image is displayed.